synapse/tools/genpkg.py

```diff
@@ -265,7 +266,11 @@ async def main(argv, outp=s_output.stdout):
     else:
         pkgdef = loadPkgProto(opts.pkgfile, opticdir=opts.optic, no_docs=opts.no_docs)
 
-    pkgdef['build'] = {'time': s_common.now()}
+    pkgdef['build'] = {
```

Comment:
I thought we were going to move this into `loadPkgProto` ?

Reply:
you're right. i forgot. will update.